src/tutorials/ping.rs

```diff
@@ -174,28 +179,32 @@
 //! [`Transport`] to the [`NetworkBehaviour`].
 //!
 //! ```rust
-//! use libp2p::{identity, PeerId, ping, Swarm};
+//! use libp2p::swarm::{keep_alive, NetworkBehaviour, Swarm};
+//! use libp2p::{identity, ping, PeerId};
 //! use std::error::Error;
 //!
 //! #[async_std::main]
 //! async fn main() -> Result<(), Box<dyn Error>> {
 //!     let local_key = identity::Keypair::generate_ed25519();
 //!     let local_peer_id = PeerId::from(local_key.public());
-//!     println!("Local peer id: {:?}", local_peer_id);
+//!     println!("Local peer id: {local_peer_id:?}");
 //!
 //!     let transport = libp2p::development_transport(local_key).await?;
 //!
-//!     // Create a ping network behaviour.
-//!     //
-//!     // For illustrative purposes, the ping protocol is configured to
-//!     // keep the connection alive, so a continuous sequence of pings
-//!     // can be observed.
-//!     let behaviour = ping::Behaviour::new(ping::Config::new().with_keep_alive(true));
-//!
-//!     let mut swarm = Swarm::with_async_std_executor(transport, behaviour, local_peer_id);
+//!     let mut swarm = Swarm::with_async_std_executor(transport, Behaviour::default(), local_peer_id);
 //!
 //!     Ok(())
 //! }
+//!
+//! /// Our network behaviour.
+//! ///
+//! /// For illustrative purposes, this includes the [`KeepAlive`](behaviour::KeepAlive) behaviour so a continuous sequence of
```

Comment:
Needs to be wrapped at 80 chars.

Reply:
On this subject, have you considered adding a `rustfmt.toml` to the project, to automate and standardize style items like this?